src/Parser/Warrior/Arms/Modules/BuffDebuff/TacticianProc.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+const debug = false;
+
+class Tactician extends Module {
```

Comment:
Class names should be the same as the file names (in this case I'd suggest renaming the item).